doc(release) Release notes collect 24.10 (#4553)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-os.mdx

## Centreon Collect

### 24.10.9

Release date: `August 4, 2025`

<details>
  <summary>Enhancements</summary>

**Broker**
- New function in the lua cache to access hostgroups aliases.

**Engine**
- [CMA] Check timeout default value has been set to 120s in agent configuration.

> - [CMA] Added token management for Agent-initiated connection.

Reviewer: ```suggestion
- [CMA] Added token management for agent-initiated connection.
```

- [configuration] a dedicated whitelist block is now available for Centreon Monitoring Agent.

</details>

<details>

>   <summary>Bug fixes</summary

Author: ```suggestion
  <summary>Bug fixes</summary>
```

> <details>
>   <summary>Enhancements</summary>
> 
> **Broker**
> - New function in the lua cache to access hostgroups alias.
> 
> **Engine**
> - [CMA] check timeout default value has been set to 120s, in Agent configuration.
> - [CMA] Added token management for Agent-initiated connection.
> - [configuration] a dedicated whitelist block is now available for Centreon Monitoring Agent.
> 
> </details>
> 
> <details>
>   <summary>Bug fixes</summary>
> 
> **Broker**
> - [Resource status] Fixed issues on tags (filters on HG, HC, SG, SC)
> 
> **Engine**
> - [Collect] Fixed a regression where an engine command could not launch several checks at the same time
> 
> </details>

Reviewer: ```suggestion
<details>
  <summary>Enhancements</summary>

**Broker**
- New function in the lua cache to access hostgroups aliases.

**Engine**
- [CMA] Check timeout default value has been set to 120s in agent configuration.
- [CMA] Added token management for Agent-initiated connection.
- [configuration] A dedicated whitelist entry is now available for Centreon Monitoring Agent.

</details>

<details>
  <summary>Bug fixes</summary>

**Broker**
- [Resource status] Fixed issues on tags (filters on HG, HC, SG, SC).

**Engine**
- [Collect] Fixed a regression where an engine command could not launch several checks at the same time.

</details>
```

## Centreon Monitoring Agent

### 24.10.7

Release date: `August 4, 2025`

<details open>
  <summary>Enhancements</summary>

- [CMA] Added a native Counter check for Windows.
- [CMA] Added a native Files check for Windows & Linux.
- [CMA] Added a native Task scheduler check for Windows & Linux.

> - [CMA] installer and CLI wording has been harmonized to fit with new UI labels.

Reviewer: ```suggestion
- [CMA] Installer and CLI wording have been harmonized to fit with new UI labels.
```

## Centreon Gorgone

### 24.10.6

Release date: `August 4, 2025`

<details>
  <summary>Bug fixes</summary>

- [Compatibility] Fixed issue preventing installation from being completed when database tables already exist.
- [Gorgone] Fixed an issue on Alma 9 where plugins were not installed because yum did not exist anymore.
- [Gorgone] Removed useless error present at every first start of gorgone.

> </details>

Reviewer: ```suggestion
<details>
  <summary>Bug fixes</summary>

- [Compatibility] Fixed an issue preventing installation from being completed when database tables already existed.
- [Gorgone] Fixed an issue on Alma 9 where plugins were not installed because yum did not exist anymore.
- [Gorgone] Removed unnecessary error present at every first start of Gorgone.

</details>
```
